src/Object/Post.php

```diff
@@ -497,6 +497,8 @@ public function getTemplateData(array $conv_responses, string $formSecurityToken
 		}
 
 		$tmp_item = [
+			'parentguid'      => $parent_guid,
+			'isreplyto'      => DI::l10n()->t('is reply to %s', $parent_username),
```

Comment:
Missing space to align the arrows.

Also no need to use `DI` in this case, the translation methods are available to all classes inheriting from `BaseModule`.
```suggestion
			'isreplyto'       => $this->t('is reply to %s', $parent_username),
```

Reply:
should i change the other usages of DI in this place too?

Reply:
Sorry... no it does not work. When i make your changes, i get a blank white page.

In friendica.log this:
```
2023-10-06T16:57:12Z app [ERROR]: Uncaught Exception Error: "Call to undefined method Friendica\Object\Post::t()" at /var/www/soc.schuerz.at/src/Object/Post.php line 501 {"exception":"Error: Call to undefined method Friendica\\Object\\Post::t() in /var/www/soc.schuerz.at/src/Object/Post.php:501\nStack trace:\n#0 /var/www/soc.schuerz.at/src/Object/Thread.php(204): Friendica\\Object\\Post->getTemplateData()\n#1 /var/www/soc.schuerz.at/src/Content/Conversation.php(685): Friendica\\Object\\Thread->getTemplateData()\n#2 /var/www/soc.schuerz.at/src/Content/Conversation.php(583): Friendica\\Content\\Conversation->getThreadList()\n#3 /var/www/soc.schuerz.at/src/Module/Item/Display.php(278): Friendica\\Content\\Conversation->render()\n#4 /var/www/soc.schuerz.at/src/Module/Item/Display.php(165): Friendica\\Module\\Item\\Display->getDisplayData()\n#5 /var/www/soc.schuerz.at/src/BaseModule.php(246): Friendica\\Module\\Item\\Display->content()\n#6 /var/www/soc.schuerz.at/src/App.php(710): Friendica\\BaseModule->run()\n#7 /var/www/soc.schuerz.at/index.php(46): Friendica\\App->runFrontend()\n#8 {main}"} - {"file":null,"line":null,"function":null,"request-id":"65203c685ec37","uid":"78691a","process_id":3269014}
```

Reply:
Sorry about that, the `Object\Post` class doesn't inherit from BaseModule and as such doesn't have the `t()` method available.